.github/workflows/testing.yml

```diff
@@ -32,10 +32,14 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
 
+      - name: Check Yarn install diffs
+        run: |
+          yarn install --ignore-scripts
```

Comment:
When it was a bash script, at least the name of the bash script's *name* explained what it does. 
I'm not familiar with 
```
    --ignore-scripts                    don't run lifecycle scripts
```
but it seems it's just skipping the automatically run scripts (e.g. `yarn run postinstall`). 
What the bash script used to accomplish was to check that the `yarn.lock` is "deterministic". I.e. running `yarn install` shouldn't trigger an edit to the `yarn.lock`. 
Ultimately, what this accomplishes is to check when people sometimes (incorrectly!) manually edits the `dependencies` (or `devDependencies`) manually rather using the `yarn add` or `yarn remove` etc.